torchsynth/module.py

```diff
@@ -150,7 +62,7 @@ def seconds_to_samples(self, seconds: T) -> T:
 
     def _forward(self, *args: Any, **kwargs: Any) -> Signal:  # pragma: no cover
         """
-        Each TorchSynthModule should override this.
+        Each TorchSynthModule0Ddeprecated should override this.
```

Comment:
why is this TorchSynthModule0Ddeprecated?

Reply:
mechanical error. I fixed in later PR but push here anyway.